packages/react-table/src/components/Table/examples/Table.md

```diff
@@ -30,7 +30,7 @@ import LeafIcon from '@patternfly/react-icons/dist/js/icons/leaf-icon';
 import FolderIcon from '@patternfly/react-icons/dist/js/icons/folder-icon';
 import FolderOpenIcon from '@patternfly/react-icons/dist/js/icons/folder-open-icon';
 
-import { ToggleGroup, ToggleGroupItem } from '@patternfly/react-core';
+import { Checkbox, ToggleGroup, ToggleGroupItem } from '@patternfly/react-core';
```

Comment:
You don't need to import anything from `@patternfly/react-core` or `@patternfly/react-table` since the theme does it for you automatically (but it doesn't hurt).